### PR TITLE
Validate FROG

### PR DESCRIPTION
**Pull request by GEployer**
# Validate [FROG](https://solscan.io/token/J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/FrogCoinSolana/status/1806767298415611956
- Coingecko/ CMC URL (If available): Coingecko applied just now

## Validation (Please check off boxes):
- [X] The metadata provided in the PR matches what is on-chain (Mandatory)
- [X] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [X] My change matches the format in the file (no spaces between fields).
- [X] My token is already live and trading on Jupiter.
- [X] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [X] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by Chefkingpin** [comment]
[X] Mint address: J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump
[X] Symbol: FROG
[X] Token Decimals: 6
[X] Token icon: https://ipfs.io/ipfs/QmbVUwQs4cw8FBW7WTR9NdT87WZrTTF1yjaRrACb1trnr3
[X] On-chain metadata matches
[X] No duplication on Jupiter's strict list

**Review by Chefkingpin — COMMENTED** [review]
[X] Mint address: J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump
[X] Symbol: FROG
[X] Token Decimals: 6
[X] Token icon: https://ipfs.io/ipfs/QmbVUwQs4cw8FBW7WTR9NdT87WZrTTF1yjaRrACb1trnr3
[X] On-chain metadata matches
[X] No duplication on Jupiter's strict list

**Comment by SchnitzelNFT** [comment]
[X] Mint address: J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump
[X] Symbol: FROG
[X] Token Decimals: 6
[X] Token icon: https://ipfs.io/ipfs/QmbVUwQs4cw8FBW7WTR9NdT87WZrTTF1yjaRrACb1trnr3
[X] On-chain metadata matches
[X] No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1828,3 +1828,4 @@ stickmanpride,$STICKY,8tgtGXhYRGKtaF4CPeVzSQpM3cUXBZ8P4ZW4hEhyLWZZ,6,https://sta
 Caprisun Monkey,capri,JD3S1oqnjG5trRTZXpLmKnS7LsKsppyFa51rPBBMWogv,6,https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f63662d697066732e636f6d2f697066732f516d4e68666b674636436b7634706964367a6d7636586b59744346655854507a70413651774267616f4172685138,true
 Zeta,ZEX,ZEXy1pqteRu3n13kdyh4LwPQknkFk3GzmMYMuNadWPo,6,https://raw.githubusercontent.com/zetamarkets/brand/master/assets/zeta.png,true
 WIFI,$WIFI,AcfKwAf4UQLA5DLRQD2eQxW6pLv79VKzVso38WMfTxGZ,6,https://ipfs.io/ipfs/QmU35RWaYbjMHWmxZjDCtDwwMGxLs29MEE7ostKj2F3iDb,true
+Frog,FROG,J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump,6,https://ipfs.io/ipfs/QmbVUwQs4cw8FBW7WTR9NdT87WZrTTF1yjaRrACb1trnr3,true
```
Comment (SchnitzelNFT) [inline validated-tokens.csv:1831]: [X] Mint address: J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump
[X] Symbol: FROG
[X] Token Decimals: 6
[X] Token icon: https://ipfs.io/ipfs/QmbVUwQs4cw8FBW7WTR9NdT87WZrTTF1yjaRrACb1trnr3
[X] On-chain metadata matches
[X] No duplication on Jupiter's strict list

**Review by Dani30Colombia — COMMENTED** [review]
[X] Mint address: J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump
[X] Symbol: FROG
[X] Token Decimals: 6
[X] Token icon: https://ipfs.io/ipfs/QmbVUwQs4cw8FBW7WTR9NdT87WZrTTF1yjaRrACb1trnr3
[X] On-chain metadata matches
[X] No duplication on Jupiter's strict list

**Review by lazim17 — COMMENTED** [review]
[X] Mint address: J57SryZGyYt3qhsa3s9HAJCfHH8h3KGtFyYbVUhPpump
[X] Symbol: FROG
[X] Token Decimals: 6
[X] Token icon: https://ipfs.io/ipfs/QmbVUwQs4cw8FBW7WTR9NdT87WZrTTF1yjaRrACb1trnr3
[X] On-chain metadata matches
[X] No duplication on Jupiter's strict list

**Comment by Jymfletzfur** [comment]
token too new, please come back in 2 weeks

**Pull request closed without merging**